script.service.ltvpl/addon.xml

```diff
@@ -0,0 +1,46 @@
+<addon id="script.service.ltvpl" name="Live TV Playlist" provider-name="jmooremcc" version="1.0.0">
+  <extension library="service.py" point="xbmc.service" />
+  <requires>
+    <import addon="xbmc.python" version="2.26.0" />
+    <import addon="script.module.enum34" version="1.1.6" />
```

Comment:
`script.module.enum34` is not available.

Reply:
I submitted the module for enum34 and no one has reviewed it and told me why it has not been accepted into the repository. The enum module is not included in the Pyton 2.7 standard library but is included in the Python 3.x standard library. My add-on requires the enum module.